.github/workflows/ci-pipeline.yml

```
@@ -19,6 +19,8 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: '16.x'
+      - name: Install mardownlint
+        run: sudo apt-get install markdownlint
```

```suggestion
        run: npm install -g markdownlint-cli
```
https://github.com/igorshubovych/markdownlint-cli#installation